src/post.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Lesson 1</title>
+    <link rel="stylesheet" href="style.css"/>
+</head>
+
+<body>
+<div class="layout">
+    <div class="header">
+        <div class="header-layout">
+            <div class="application-logo"></div>
+            <div class="application-text">
+                <div class="application-text__greeting">Forget about skirts, their time has gone...</div>
+                <div class="application-text__cta">Leggings rules the world!</div>
+            </div>
+            <div class="user-menu"></div>
+        </div>
+    </div>
+
+    <div class="content">
+        <div class="selected-post"></div>
+        <div class="related-posts__list"></div>
+    </div>
```

Comment:
BEM is something I'm not good in.
I've just added class names similar to (as I saw it) posts.html class names:
```
.content -1> .selected-post -1> .post-preview && .selected-post__post -1> .post-preview__island -1> .post-preview__img-wrapper -1> .post-preview__img
.content -1> .selected-post -1> .post-preview && .selected-post__post -1> .post-preview__island -1> .post-preview__description
.content -1> .selected-post -1> .selected-post__comments -0..n> .selected-post__comment
.content -1> .related-posts__list -0..n> .post-preview && .related-posts__post -1> .post-preview__island -1> .post-preview__img-wrapper -1> .post-preview__img
.content -1> .related-posts__list -0..n> .post-preview && .related-posts__post -1> .post-preview__island -1> .post-preview__island -1> .post-preview__description
```

I think I'll read more about BEM and totally rebuild this page with some modularisation lesson.